lang/fr.json

```diff
@@ -384,7 +385,7 @@
   "CreateOrder.TitleForEvent": "Commander des tickets pour {event}",
   "createPledge.collectiveDetails": "Détails du nouveau collectif :",
   "createPledge.collectiveURL": "URL du collectif",
-  "createPledge.conditions": "Vous ne pouvez pour l'instant faire de promesse de dons qu'aux projets qui disposent d'un dépôt ou d'un profil d'organisation sur GitHub. Le projet doit avoir au moins 100 étoiles !",
+  "createPledge.conditions": "At the moment, you can only pledge for Open Source projects with a GitHub repository or organization. We\n                request the project to have a least 100 stars on GitHub!",
```

Comment:
I don't understand why some translations were removed, maybe a problem with the tools in some specific operating system. We should investigate and reset them to what they were.

Reply:
How do I do this please?

Reply:
I synced my this branch with the upstream/master, but it still did not fix this issue